tools/travis/build.sh

```diff
@@ -11,7 +11,7 @@ tools/build/scanCode.py .
 
 cd $ROOTDIR/ansible
 
-ANSIBLE_CMD="ansible-playbook -i environments/travis"
+ANSIBLE_CMD="ansible-playbook -i environments/local"
```

Comment:
`travis` environment is no longer needed. Travis changed the docker gateway to be `172.17.0.1` so I figured, having two identical environments feels wrong.

`local` simply means: install to the local docker daemon which i think is fine.